app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
@gaganawhad okay as I think more about this, I think we have more to discuss.

GA and FB differ in a very important way: GA _must_ be one API call per location. FB can bundle multiple locations into a single API call.

This appears to have led to a pair of divergent design directions:

* Every GA job updates a single location.
* Every FB job updates multiple locations.

This is fine, as far as it goes. BUT, you've taken the "FB job updates multiple locations" to the extreme of "FB job updates _all_ locations."

I think lots of jobs is preferable to a single job because then our workers can execute them all in parallel, and we can scale the workers if needed. As it is, we have to iterate through them all sequentially.